enhancements/single-node/single-node-openshift-with-workers.md

```diff
@@ -0,0 +1,509 @@
+---
+title: single-node-openshift-with-workers
+authors:
+  - "@omertuc"
+reviewers:
```

Comment:
Restoring some comments from the previous PR that got closed:

> romfreiman 2 days ago

@sinnykumari PTAL

_____________________________

> romfreiman 2 days ago

@dhellmann @sdodson who should we add as approvers?
_____________________________
 
 >dhellmann 2 days ago
 
There's an API change here, so we need an API approver to look at it. @mfojtik can you help identify someone with time to review this?

I would expect to see someone from the networking team familiar with ingress, too. @knobunc can you help find a reviewer?
_____________________________
 
> romfreiman 8 hours ago

@tkashem
 _____________________________

> romfreiman 2 days ago

@Miciah who's the approver from network-edge side?